javascript/event-recommender-part2.md

```diff
@@ -43,10 +43,12 @@ Now that you've finished Part 1 and your Event Recommender class is complete, yo
 
 ### Instructions for adding tests
 
-1. Go back to your original JavaScript file where you defined your EventRecommender, Event, and User classes. Refamiliarize yourself with what you did here. We will be adding *Jasmine tests* for each function defined in your EventRecommender. In the bottom of this file, add the line: `module.exports = EventRecommender;`
+1. Go back to your original JavaScript file where you defined your EventRecommender, Event, and User classes. Refamiliarize yourself with what you did here. We will be adding *Jasmine tests* for each function defined in your EventRecommender. In the bottom of your EventRecommender file, add the line: `module.exports = EventRecommender;`
```

Comment:
If we’re being explicit, can you change this to the actual file name?

Reply:
I am unsure of what you are referring to.

Reply:
instead of "your EventRecommender file", can you just add the name of the file to be extra clear?

Reply:
Yes of course, I understand now.